static/elements/chromedash-upcoming.js

```diff
@@ -0,0 +1,95 @@
+import {LitElement, html} from 'lit-element';
+import '@polymer/iron-icon';
+import style from '../css/elements/chromedash-upcoming.css';
+
+const TEMPLATE_CONTENT = {
+  stable: {
+    channelLabel: 'Stable',
+    h1Class: '',
+    downloadUrl: 'https://www.google.com/chrome/browser/desktop/index.html',
+    downloadTitle: 'Download Chrome stable',
+    dateText: 'was',
+    featureHeader: 'Features in this release',
+  },
+  beta: {
+    channelLabel: 'Next up',
+    h1Class: 'chrome_version--beta',
+    downloadUrl: 'https://www.google.com/chrome/browser/beta.html',
+    downloadTitle: 'Download Chrome Beta',
+    dateText: 'between',
+    featureHeader: 'Features planned in this release',
+  },
+  dev: {
+    channelLabel: 'Dev',
+    h1Class: 'chrome_version--dev',
+    downloadUrl: 'https://www.google.com/chrome/browser/canary.html',
+    downloadTitle: 'Download Chrome Canary',
+    dateText: 'coming',
+    featureHeader: 'Features planned in this release',
+  },
+};
+const REMOVED_STATUS = ['Removed'];
+const DEPRECATED_STATUS = ['Deprecated', 'No longer pursuing'];
```

Comment:
Granting that I don't know how this constant is being used, 'deprecated' and 'no longer pursuing' aren't the same thing.

Reply:
Hmm... This constant is being used to display the tooltip and icon for 'deprecated' and 'no longer pursuing' features on the CURRENT (OLD) releases page. The developers, who developed the OLD releases page, used the same icon and tooltip for both the deprecated and no longer pursuing features. Hence, it  may be the reason for them to define the constant this way. 

Anyway, Is it correct and helpful for the users that the same icon be used for both the types of feature on the releases page?

Reply:
I think this is a problem. If I'm scrolling through the list with everything, this icon is the only thing that tells me the status of a feature. It provides incorrect information.

@jrobbins should I open a ticket on this?

Reply:
Would a feature that is no longer pursed belong on the roadmap view at all?

I'm thinking that this might be code that was copied-and-pasted a while back from the feature list page, where such a feature might appear in the section of the feature list for those features.  However, this code is only for the "upcoming" roadmap view.

If this logic case is needed for this page, then let's track a separate issue for it.

Reply:
@cwilso Please comment. I believe there is a reason to display this. The one that occurs to me is to have a record of things considered and rejected so that later engineers don't re-propose the same or similar ideas. I wonder if we should go so far as to require an explanation for why something is no longer being pursued. It might save someone from starting down a dead-end road.

Reply:
Features that are no-longer-pursued would still be in the feature list.  I just don't see a place for them on the "Upcoming" page because they are not part of any upcoming release.